src/kOS.Safe/Execution/CPU.cs

```diff
@@ -1074,19 +1115,16 @@ public void KOSFixedUpdate(double deltaTime)
                 {
                     ProcessTriggers();
 
-                    if (currentStatus == Status.Running || currentStatus == Status.Waiting)
+                    if (showStatistics)
                     {
-                        if (showStatistics)
-                        {
-                            executionWatch.Start();
-                            instructionWatch.Start();
-                        }
-                        ContinueExecution(showStatistics);
-                        numMainlineInstructions = instructionsSoFarInUpdate;
-                        if (showStatistics)
-                        {
-                            executionWatch.Stop();
-                        }
+                        executionWatch.Start();
+                        instructionWatch.Start();
+                    }
+                    ContinueExecution(showStatistics);
+                    numMainlineInstructions = instructionsSoFarInUpdate;
+                    if (showStatistics)
+                    {
+                        executionWatch.Stop();
```

Comment:
The above is just git diff's messy way of showing that I out-dented this code when I removed the wrapping if-condition around it.